app/presenters/blacklight/show_presenter.rb

```diff
@@ -35,9 +35,9 @@ def html_title
         fields = Array.wrap(view_config.html_title_field)
         f = fields.detect { |field| document.has? field }
         f ||= 'id'
-        field_values(field_config(f))
+        field_values(field_config(f), no_html: true)
       else
-        heading
+        heading(no_html: true)
```

Comment:
This bothers me a little bit because it isn't actually used in blacklight.. It almost makes me wonder if we're appropriately using `field_values` here at all. I'd like to put a little more thought into it before we ship this.